usagov/vote-gov-drupal
# VOTE-896 Apply styling to the basic page markup

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-896

## Description (optional)
Apply the USWDS vote styles to the basic page type

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. recompile theme assets npm run build
2. run lando drush cr

### Testing steps
1. Login as an admin and create a test basic page.
2. View the test basic page in the browser and make sure all styles are applied 
    a. Desktop
    b. Mobile
